### DISCUSSION TITLE
Nextjs 11 Script

### DISCUSSION THREAD

**Discussion by wpcodevo** [vercel/next.js#26733] [Help]
I used nextjs 11 `<Script />` for google ads and google platform scripts but the scripts are not executed. When I change it to the normal `<script/>` then all the scripts are executed.
`<Script src='//pagead2.googlesyndication.com/pagead/js/adsbygoogle.js' />
          <Script
            src={`https://www.googletagmanager.com/gtag/js?id=${GA_TRACKING_ID}`}
          />
          <Script
            src='https://apis.google.com/js/platform.js'
            onLoad={() => {
              console.log("Loaded successfully Codevo");
            }}
          />
          <Script
            dangerouslySetInnerHTML={{
              __html: `
            window.dataLayer = window.dataLayer || [];
            function gtag(){dataLayer.push(arguments);}
            gtag('js', new Date());
            gtag('config', '${GA_TRACKING_ID}', {
              page_path: window.location.pathname,
            });
          `,
            }}
          />`

**Comment by wpcodevo** [comment]
> 
> 
> I used nextjs 11 `<Script />` for google ads and google platform scripts but the scripts are not executed. When I change it to the normal `<script/>` then all the scripts are executed.
> `<Script src='//pagead2.googlesyndication.com/pagead/js/adsbygoogle.js' /> <Script src={`[https://www.googletagmanager.com/gtag/js?id=${GA_TRACKING_ID}`}](https://www.googletagmanager.com/gtag/js?id=$%7BGA_TRACKING_ID%7D%60%7D)
> />
> <Script
> src='https://apis.google.com/js/platform.js'
> onLoad={() => {
> console.log("Loaded successfully Codevo");
> }}
> />
> <Script
> dangerouslySetInnerHTML={{
> __html: `window.dataLayer = window.dataLayer || []; function gtag(){dataLayer.push(arguments);} gtag('js', new Date()); gtag('config', '${GA_TRACKING_ID}', { page_path: window.location.pathname, });`,
> }}
> />`

I figured it out

**Reply by phuoccss** [reply]
Hi. How did you make it work? I'm using inline script with beforeInteractive strategy and it's not working.
